src/main/ml-modules/root/client/app/account/login/login.controller.js

```diff
@@ -17,9 +17,12 @@ angular.module('demoApp')
         .then( function() {
           $http.get('/api/checkTemplates').success(function(data, status, headers, config) {
             if (status == 200) {
-              if(!data.queryTemplateExists && data.isSearchUser) {
-                Auth.homeMessage = "There are no queries to search. Please contact the Data Explorer admin (Wizard User) to create queries";
-                Auth.homeMessageClass = "alert alert-warning";
+              if(!data.queryTemplateExists) {
+                if(data.isSearchUser) {
+                  Auth.homeMessage = "<span class=\"alert alert-warning\">There are no queries to search. Please contact the Data Explorer admin (Wizard User) to create queries.</span>";
```

Comment:
People may not know about the ML security role of wizard user - suggest rewording to

 "Please contact a query maintainer who can add new queries that will show here. These users are anyone configured with the 'wizard-user' role in MarkLogic."

Reply:
Good call. I've pushed the suggested rewording.